lib/ts/recipeImplementation.ts

```diff
@@ -137,6 +144,17 @@ export default function RecipeImplementation(recipeImplInput: {
             }
 
             // we do not send an event here since it's triggered in setIdRefreshToken area.
+        },
+
+        async getInvalidClaimsFromResponse(response: AxiosResponse | Response): Promise<ClaimValidationError[]> {
```

Comment:
Can you make the function signature of this similar to other functions in this file?

Reply:
I can, but a `userContext` doesn't make sense here, it'd be always undefined and not reused.

Reply:
Also need to expose this function via the index file so that the user can use it